CHANGELOG.md

```diff
@@ -2,160 +2,178 @@
+  OCI-mode by default. It can be negated with a new `--no-oci` command line flag.
+
+See the [admin guide](https://docs.sylabs.io/guides/latest/admin-guide/) and
+[user guide](https://docs.sylabs.io/guides/latest/user-guide/) for full
+requirements of OCI-mode and usage information.
+
 ### Changed defaults / behaviours
 
-- When building RPM, we will now use `/var/lib/singularity` (rather than
-  `/var/singularity`) to store local state files.
-- `--cwd` is now the preferred form of the flag for setting the container's
-  working directory, though `--pwd` is still supported for compatibility.
-- The way --home is handled when running as root (e.g. `sudo singularity`) or
-  with `--fakeroot` has changed. Previously, we were only modifying the `HOME`
-  environment variable in these cases, while leaving the container's
-  `/etc/passwd` file unchanged (with its homedir field pointing to `/root`,
-  regardless of the value passed to `--home`). With this change, both value of
-  `HOME` and the contents of `/etc/passwd` in the container will reflect the
-  value passed to `--home`.
-- Bind mounts are now performed in the order of their occurrence on the command
-  line, or within the value of the `SINGULARITY_BIND` environment variable.
-  (Previously, image-mounts were always performed first, regardless of order.)
-- Support for image driver plugins, deprecated at 3.11, has been removed.
-  Unprivileged kernel overlay is supported without a plugin. In
-  `singularity.conf`, the `image driver` directive has been removed, and
-  `enable overlay` no longer supports the `driver` option.
+#### Packages / Requirements
+
+- RPM packages now use `/var/lib/singularity` (rather than `/var/singularity`)
+  to store local state files.
 - Bash completions are now install to the modern
   `share/bash-completion/completions` location, rather than under `etc`.
-- Default OCI config generated with `singularity mount` no longer sets any
-  inheritable / ambient capabilites, matching other OCI runtimes.
-- In OCI-mode, a container run as root, or with `--fakeroot` has OCI default
-  effective/permitted capabilities.
-- Lookup and store user/group information in stage one prior to entering any
-  namespaces to fix issue with winbind not correctly lookup user/group
-  information when using user namespace.
-- `singularity oci mount` now uses, and requires, `squashfuse` to mount a SIF
-  image to an OCI bundle.
 - The `--vm` and related flags to start singularity inside a VM have been
   removed. This functionality was related to the retired Singularity Desktop /
   SyOS projects.
+- Singularity uses squashfuse_ll/squashfuse, which is now built from a git
+  submodule unless `--without-squashfuse` is specified as an argument to
+  `mconfig`. When built with `--without-squashfuse`, `squashfuse_ll` or
+  `squashfuse` will be located on `PATH`. Version 0.2.0 or later is required.
 - The keyserver-related commands that were under `remote` have been moved to
```

Comment:
Is it intentional that this (`keyserver`) stayed here while its "brothers" (`registry` and `remote`) are in the "CLI" subsection?

Reply:
nope.. will move